src/porepy/models/incompressible_flow_model.py

```diff
@@ -224,6 +237,25 @@ def _assign_variables(self) -> None:
                     self.mortar_variable: {"cells": 1},
                 }
 
+    def _create_dof_and_eq_manager(self) -> None:
```

Comment:
I have tried and the non-AD case is not supported, my bad. I thought non-AD was supported since in the `after_newton_convergence` there existed an instance of the assembler object being used. Now any instance of the assembler object has been removed. The methods are now documented.